backend/src/db/migrations/20230915120000_add_deleted_at_to_users_table.ts

```diff
@@ -0,0 +1,18 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'users';
+const COLUMN_NAME = 'deleted_at';
+
+async function up(knex: Knex): Promise<void> {
+  await knex.schema.table(TABLE_NAME, (table) => {
+    table.dateTime(COLUMN_NAME).nullable();
```

Comment:
it's nullable by default
```suggestion
    table.dateTime(COLUMN_NAME);
```